<launch>

  <!-- node name="rqt_console" pkg="rqt_console" type="rqt_console" / -->

  <group ns="jaco">
    <node name="jaco_driver" pkg="jaco_driver" type="jaco_arm_driver" output="screen" cwd="node" >
    </node>
    
    <!-- node name="jaco_tf_updater" pkg="jaco_driver" type="jaco_tf_updater" output="screen" cwd="node">
      <remap from="in/joint_angles" to="out/joint_angles"/>
    </node -->
  </group>
</launch>
